control-plane/config/crd/patches/webhook_in_terminatinggatewayservices.yaml

```diff
@@ -0,0 +1,17 @@
+# The following patch enables conversion webhook for CRD
```

Comment:
This file can be removed when we remove the patches directory

Reply:
Done